lib/logging/logging.dart

```diff
@@ -92,7 +92,7 @@ class LoggingScreen extends Screen {
   }
 
   void _updateStatus() {
-    final int count = loggingTable.rows.length;
+    final int count = loggingTable.data.length;
```

Comment:
Perhaps expose a `rowCount` getter on `Table`?

Reply:
Good call :)

Done, and also moved the constructors to fix the lint. Running "Sort Members" on these files makes a LOT of changes - we may want to do it at some point when there aren't too many PRs open!